docker/README.md

```diff
@@ -241,3 +303,23 @@ or
 ```bash
 curl -X GET http://your_public_host_name:8081/ping
```

Comment:
:80 / no port ought to work for docker container

Reply:
Corrected this.